src/modules/transaction/components/TransactionRow/index.test.js

```diff
@@ -17,25 +18,25 @@ const txBase = {
   id: 'ad0e0acbe8a3ece3087c8362149ca39c470e565d268df32e57de5d3fe2e1ea5c',
 };
 
-const vote = {
-  moduleCommandID: '5:1',
-  params: {
-    votes: [
-      {
-        amount: '2000000000',
-        delegateAddress: accounts.delegate.summary.address,
-      },
-    ],
-  },
-};
-const registerDelegate = {
-  moduleCommandID: '5:0',
-  params: {
-    delegate: {
-      username: 'sample_username',
-    },
-  },
-};
+// const vote = {
+//   moduleCommandID: '5:1',
+//   params: {
+//     votes: [
+//       {
+//         amount: '2000000000',
+//         delegateAddress: accounts.delegate.summary.address,
+//       },
+//     ],
+//   },
+// };
+// const registerDelegate = {
+//   moduleCommandID: '5:0',
+//   params: {
+//     delegate: {
+//       username: 'sample_username',
+//     },
+//   },
+// };
```

Comment:
@eniolam1000752 If these are no longer in use, why are they commented out instead of removed? Please add a TODO if they're to be fixed at a later time